pages/osx/nettop.md

```diff
@@ -0,0 +1,35 @@
+# nettop
+
+> Display dynamic real-time information about the network.
+
+- Start nettop monitoring TCP and UDP sockets from all interfaces:
+
+`nettop`
+
+- Start nettop monitoring TCP sockets from Loopback interfaces:
+
+`nettop -m tcp -t loopback`
+
+- Start nettop monitoring specific process:
```

Comment:
```suggestion
- Start nettop monitoring a specific process:
```

Reply:
Does it focus on 1 process, if not then

```suggestion
- Start nettop, monitoring specific processes:
```
would be acceptable

Reply:
Yes, it focus on 1 process, I should add `a` :)